### PR TITLE
Improving test cases for more invalid revocation requests.

### PR DESCRIPTION
**Pull request by Pushpalanka**

### CODE REVIEW

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2TokenRevokeWithInvalidClientCredentialsTestCase.java [hunk 1]
```diff
@@ -57,26 +62,57 @@ public void testInit() throws Exception {
...
+                        + "error code");
+
+        //client credentials been null
+        errorMessage = revokeToken(token, null);
+        Assert.assertEquals(OAuth2Constant.INVALID_CLIENT, errorMessage,
+                "Invalid format in sending client credentials, should have produced : " + OAuth2Constant.INVALID_CLIENT
+                        + "error code");
+
+        //revoke with invalid client key
+        errorMessage = revokeToken(token, "dummyConsumerKey" + consumerSecret);
```
Comment (GayanM) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2TokenRevokeWithInvalidClientCredentialsTestCase.java]: base64 encode ?

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2TokenRevokeWithInvalidClientCredentialsTestCase.java [hunk 2]
```diff
@@ -48,6 +50,9 @@
 public class OAuth2TokenRevokeWithInvalidClientCredentialsTestCase extends OAuth2ServiceAbstractIntegrationTest {
 
     private static final String REVOKE_TOKEN_API_ENDPOINT = "https://localhost:9853/oauth2/revoke";
```
Comment (dimuthud) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2TokenRevokeWithInvalidClientCredentialsTestCase.java]: Can we get these ips & port values using automation.xml file ?
https://localhost:9853/oauth2/token

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2TokenRevokeWithInvalidClientCredentialsTestCase.java [hunk 3]
```diff
@@ -48,6 +50,9 @@
...
+    private static final String TOKEN_API_ENDPOINT = "https://localhost:9853/oauth2/token";
```
Comment (dimuthud) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2TokenRevokeWithInvalidClientCredentialsTestCase.java]: Can we get these ips & port values using automation.xml file ?
https://localhost:9853/oauth2/token

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2TokenRevokeWithInvalidClientCredentialsTestCase.java [hunk 4]
```diff
@@ -57,26 +62,63 @@ public void testInit() throws Exception {
...
+        //revoke with invalid client secret
+        errorMessage = revokeToken(token, getBase64EncodedString(consumerKey, "dummyConsumerSecret"));
+        Assert.assertEquals(OAuth2Constant.UNAUTHORIZED_CLIENT, errorMessage,
+                "Invalid consumer secret in sending client credentials, should have produced : " + OAuth2Constant.INVALID_CLIENT
+                        + "error code");
+
+        //token is required parameter in request
+        errorMessage = revokeToken(null, getBase64EncodedString(consumerKey, consumerSecret));
+        //The request is missing a required parameter as per spec.
+        Assert.assertEquals(OAuth2Constant.INVALID_REQUEST, errorMessage,
```
Comment (dimuthud) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2TokenRevokeWithInvalidClientCredentialsTestCase.java]: We can static import of assetEquals() you dont need to call  Assert.assertEquals()

**Comment by JKAUSHALYA** [comment]
Closing until conflicts are resolved.

**Pull request closed without merging**